DetectorDescription/DDCMS/interface/DDSolidShapes.h

```diff
@@ -100,7 +113,9 @@ namespace cms {
        {DDSolidShape::ddellipticaltube, "EllipticalTube"},
        {DDSolidShape::ddcuttubs, "CutTube"},
        {DDSolidShape::ddextrudedpolygon, "ExtrudedPolygon"},
-       {DDSolidShape::ddtrd1, "Trd1"}}};
+       {DDSolidShape::ddtrd1, "Trd1"},
+       {DDSolidShape::ddunsupported1, "Unsupported1"},
```

Comment:
why is it needed?

Reply:
I put the old set back (including the old enum)